CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Snyk Code: add support for Single Tenant setups
+- Vulnerability detail views now have links to Snyk Learn when we have an appropriate lesson available.
```

Comment:
Can you create a separate 1.2.14 entry for this please? This change won't go into 1.2.13 since this it be released today.